org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/services/extensions/XMLExtensionRegistryTest.java

```diff
@@ -0,0 +1,83 @@
+package org.eclipse.lemminx.services.extensions;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.function.Predicate;
+
+import org.eclipse.lemminx.XMLAssert.SettingsSaveContext;
+import org.eclipse.lemminx.XMLLanguageServer;
+import org.eclipse.lemminx.dom.DOMDocument;
+import org.eclipse.lemminx.services.XMLLanguageService;
+import org.eclipse.lemminx.services.extensions.IXMLExtension;
+import org.eclipse.lemminx.services.extensions.XMLExtensionsRegistry;
+import org.eclipse.lemminx.services.extensions.save.ISaveContext;
+import org.eclipse.lsp4j.InitializeParams;
+import org.junit.jupiter.api.Test;
+
+import com.google.gson.JsonObject;
+
+public class XMLExtensionRegistryTest {
```

Comment:
To be consistent with existing test ExtensionRegistryDisposeTest, I suggest to rename this class with ExtensionRegistryDoSaveTest